groups.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_security_group" "this" {
+  name_prefix = var.name
+  description = "Attached to all Netbox instances"
+  vpc_id      = var.vpc_id
+
+  tags = merge(
+    var.tags,
+    map(
+      "Name", var.name
+    )
```

Comment:
Is there a reason you went with `map()` syntax instead of `{}` syntax?

Reply:
not a good one